### PR TITLE
Allow bindings to handle unknown objects in trace_object()

### PR DESCRIPTION
**Pull request by qinsoon**
This PR adds a new method to `ActivePlan` to allow bindings to handle unknown objects encountered in object tracing.

### CODE REVIEW

File: src/policy/space.rs [hunk 1]
```diff
@@ -181,8 +181,9 @@ impl SFT for EmptySpaceSFT {
         _object: ObjectReference,
         _worker: GCWorkerMutRef,
     ) -> ObjectReference {
+        // We do not have the `VM` type parameter here, so we cannot forward the call to the VM.
         panic!(
-            "Call trace_object() on {} (chunk {}), which maps to an empty space",
+            "Call trace_object() on {} (chunk {}), which maps to an empty space. SFTProcessEdges does not support the fallback to vm_trace_object().",
             _object,
```
Comment (wks) [inline src/policy/space.rs:187]: This `_object` has the same problem.  Although it is not related to the bindings, it is nice to fix it, too.

**Review by wks — CHANGES_REQUESTED** [review]
The logic is OK.

There is only a small problem related to the naming of used/unused parameters.

**Comment by qinsoon** [comment]
I have removed the underscore prefix for those variables.

**Review by wks — APPROVED** [review]
LGTM

**Pull request merged**